docs/about.md

```diff
@@ -2,25 +2,44 @@
 
 We have a [GitHub discussions](https://github.com/OpenMathLib/OpenBLAS/discussions/) forum to discuss usage and development of OpenBLAS. We also have a [Google group for *users*](https://groups.google.com/forum/#!forum/openblas-users) and a [Google group for *development of*](https://groups.google.com/forum/#!forum/openblas-dev) OpenBLAS. 
 
-## Donations
-
-You can read OpenBLAS statement of receipts and disbursement and cash balance on [google doc](https://docs.google.com/spreadsheet/ccc?key=0AghkTjXe2lDndE1UZml0dGpaUzJmZGhvenBZd1F2R1E&usp=sharing). A backer list is available [on GitHub](https://github.com/OpenMathLib/OpenBLAS/blob/develop/BACKERS.md).
-
-We welcome the hardware donation, including the latest CPU and boards.
-
 ## Acknowledgements
 
-This work is partially supported by
+This work was or is partially supported by the following grants, contracts and institutions:
+
 * Research and Development of Compiler System and Toolchain for Domestic CPU, National S&T Major Projects: Core Electronic Devices, High-end General Chips and Fundamental Software (No.2009ZX01036-001-002)
 * National High-tech R&D Program of China (Grant No.2012AA010903)
+* Chan Zuckerberg Initiative's Essential Open Source Software for Science program:
+    * Cycle 1 grant: [Strengthening NumPy's foundations - growing beyond code](https://figshare.com/articles/journal_contribution/Proposal_NumPy_OpenBLAS_for_Chan_Zuckerberg_Initiative_EOSS_2019_round_1/10302167) (2019-2020)
+    * Cycle 3 grant: [Improving usability and sustainability for NumPy and OpenBLAS](https://chanzuckerberg.com/eoss/proposals/improving-usability-and-sustainability-for-numpy-and-openblas/) (2020-2021)
+* Sovereign Tech Fund funding: [Keeping high performance linear algebra computation accessible and open for all](https://www.sovereigntechfund.de/tech/openblas) (2023-2024)
+
+Over the course of OpenBLAS development, a number of donations were received.
+You can read OpenBLAS's statement of receipts and disbursement and cash balance in
+[this Google doc](https://docs.google.com/spreadsheet/ccc?key=0AghkTjXe2lDndE1UZml0dGpaUzJmZGhvenBZd1F2R1E&usp=sharing) (covers 2013-2016).
+A list of backers is available [in BACKERS.md](https://github.com/OpenMathLib/OpenBLAS/blob/develop/BACKERS.md) in the main repo.
+
+### Donations
+
+We welcome hardware donations, including the latest CPUs and motherboards.
+
 
 ## Users of OpenBLAS
```

Comment:
I believe the (short and mixed) original list here is the result of a user survey. Not sure if we want to kick out less-well known projects, and not sure if we want to group by prominence (which might vary with the profession of the user reading this). 
In that spirit, "Known open source users..." might be a more appropriate heading (and as far as I know R may still not be using OpenBLAS by default, only through a popular add-in) ?

Reply:
The two links I removed are for projects that are dead, which seemed like a good reason to remove them. And Ceemple turned into OpenCV Ceemple, and then the separate name went away so whatever that was in the past is now simply part of OpenCV (which explains why I added OpenCV).

> In that spirit, "Known open source users..." might be a more appropriate heading (and as far as I know R may still not be using OpenBLAS by default, only through a popular add-in) ?

Sure will change. And yes, not sure - may also depend on the R install method, there's more than one way.

Reply:
ok, if they're dead that does seem like a perfect reason for removal :)